src/app/RecordingMetadata/ClickableLabel.tsx

```diff
@@ -16,47 +16,47 @@
 
 import { Label } from '@patternfly/react-core';
 import React from 'react';
-import { RecordingLabel } from './RecordingLabel';
+import { RecordingLabel } from './types';
 
 export interface ClickableLabelCellProps {
   label: RecordingLabel;
   isSelected: boolean;
   onLabelClick: (label: RecordingLabel) => void;
 }
 
-export const ClickableLabel: React.FC<ClickableLabelCellProps> = ({ onLabelClick, ...props }) => {
+export const ClickableLabel: React.FC<ClickableLabelCellProps> = ({ label, isSelected, onLabelClick }) => {
   const [isHoveredOrFocused, setIsHoveredOrFocused] = React.useState(false);
-  const labelColor = React.useMemo(() => (props.isSelected ? 'blue' : 'grey'), [props.isSelected]);
+  const labelColor = React.useMemo(() => (isSelected ? 'blue' : 'grey'), [isSelected]);
 
   const handleHoveredOrFocused = React.useCallback(() => setIsHoveredOrFocused(true), [setIsHoveredOrFocused]);
   const handleNonHoveredOrFocused = React.useCallback(() => setIsHoveredOrFocused(false), [setIsHoveredOrFocused]);
 
   const style = React.useMemo(() => {
     if (isHoveredOrFocused) {
       const defaultStyle = { cursor: 'pointer', '--pf-c-label__content--before--BorderWidth': '2.5px' };
-      if (props.isSelected) {
+      if (isSelected) {
         return { ...defaultStyle, '--pf-c-label__content--before--BorderColor': '#06c' };
```

Comment:
Maybe these things can also be extracted to consts?

Reply:
Would u prefer it to be a separate issue/PR to clean up the overall css?

Reply:
https://github.com/cryostatio/cryostat-web/issues/853